client/src/elm/Translate.elm

```diff
@@ -8446,6 +8729,33 @@ translationSet trans =
                     , kinyarwanda = Just "Nta kimenyetso na kimwe"
                     }
 
+        Predecessor predecessor ->
+            case predecessor of
+                PredecessorFather ->
+                    { english = "Father"
+                    , kinyarwanda = Nothing
```

Comment:
How do we handle translation flow? Can we export not yet translated strings to translators from code at a later time?

Reply:
@AronNovak 
We do not hold deployments till everything is translated.
Translations are completed with follow up PRs.